pkg/types/ovirt/defaults/platform.go

```diff
@@ -1,11 +1,80 @@
 package defaults
 
 import (
+	"fmt"
+
+	"github.com/ovirt/go-ovirt"
+
+	ovirtinstallconfig "github.com/openshift/installer/pkg/asset/installconfig/ovirt"
 	"github.com/openshift/installer/pkg/types/ovirt"
 )
 
+// DefaultNetworkName is the default network name to use in a cluster
+const DefaultNetworkName = "ovirtmgmt"
+
 // SetPlatformDefaults sets the defaults for the platform.
-func SetPlatformDefaults(p *ovirt.Platform) {
-	// no platform defaults
-	return
+func SetPlatformDefaults(p *ovirt.Platform) error {
+	con, err := ovirtinstallconfig.NewConnection()
+	if err != nil {
+		return err
+	}
+	defer con.Close()
+
+	setDefaultNetwork(p)
+
+	err = setDefaultVNICProfile(p, con)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func setDefaultNetwork(p *ovirt.Platform) {
+	if p.NetworkName == "" {
+		p.NetworkName = DefaultNetworkName
+	}
+}
+
+func setDefaultVNICProfile(p *ovirt.Platform, con *ovirtsdk.Connection) error {
+	if p.VNICProfileID == "" {
+		clusterResponse, err := con.SystemService().ClustersService().ClusterService(p.ClusterID).Get().Follow("networks").Send()
```

Comment:
As I said in my other comment, any defaults added here need to be static values. For one, I think commands like `create manifests` or `create ignition-configs` should be executable without access to the rhv api. API access like this in general should live in `pkg/asset` but I'm not aware of a similar example where default values are set.

Reply:
> ~~For one, I think commands like `create manifests` or `create ignition-configs` should be executable without access to the rhv api.~~

I was wrong about this, so I updated. There are examples in AWS where installconfig is validated with API connections.